### ISSUE TITLE
[unity] Timeline bug when call PlayableDirector.pause

### ISSUE THREAD

**Issue by k-okawa** [EsotericSoftware/spine-runtimes#1980]
com.esotericsoftware.spine.timeline-4.0

SpineAnimationStateMixerBehaviour.cs
``` csharp
public override void OnGraphStop (Playable playable) {
	if (endAtClipEnd)
		HandleClipEnd();
	else if (isPaused) // stop event occurred after pause, so resume again
		HandleResume(playable);
}
```

As in the comment, OnGraphStop is called after pause.

If endAtClipEnd is true,  HandleClipEnd is called after pause,
so clip end when PlayableDirector.Pause called.

If endAtClipEnd is false and isPaused is true, HandleResume called after pause,
so does not pause clip when PlayableDirector.Pause called.

I think correct code is below
``` csharp
public override void OnGraphStop (Playable playable) {
	if (!isPaused && endAtClipEnd)
		HandleClipEnd();
}
```

I'm sorry I'm not good at English.

**Comment by HaraldCsaszar** [comment]
Thanks very much for reporting and for providing a bugfix up front! After performing some necessary tests we have released new 4.0 and 4.1-beta Spine Timeline UPM packages. They are available for download here as usual:
http://esotericsoftware.com/spine-unity-download#Download

**Comment by k-okawa** [comment]
Thank you for release!
